c2corg_api/search/notify_sync.py

```diff
@@ -0,0 +1,40 @@
+import transaction
+import logging
+from kombu.pools import producers
+
+
+log = logging.getLogger(__name__)
+
+
+def notify_es_syncer(queue_config):
+    """Notify the syncer script that a document has changed by pushing a
+    friendly message into the Redis queue.
+    """
+    def push_notification():
+        with producers[queue_config.connection].acquire(
+                block=True, timeout=3) as producer:
```

Comment:
An Empty exception will be [thrown](https://docs.python.org/3/library/queue.html#queue.Queue.get) and an error message is logged. I wonder if that is enough.